cli/cmdlineparser.cpp

```diff
@@ -1590,11 +1585,27 @@ CmdLineParser::Result CmdLineParser::parseFromArgs(int argc, const char* const a
         return Result::Fail;
     }
 
+    for (auto& path : mIgnoredPaths)
+    {
+        path = Path::removeQuotationMarks(std::move(path));
+        path = Path::simplifyPath(std::move(path));
+
+        bool isdir = false;
+        if (!Path::exists(path, &isdir) && mSettings.debugignore)
+            std::cout << "path to ignore does not exist: " << path << std::endl;
```

Comment:
is there a better way to output this. A plugin might expect "cppcheck: error: ..." if we are fortunate but it will not expect this output :-(
if it at least starts with "cppcheck: " that is better imho.

Reply:
None of any of the debug messages are currently prefixed. That also needs to go through a dedicated logger/stream but is currently not the case and will probably be some work. That is something to *maybe* look into *after* https://trac.cppcheck.net/ticket/13200 is fixed.